### DISCUSSION TITLE
Pro Micro SPI with MAX7219SevenSegmentDisplay

### DISCUSSION THREAD

**Discussion by simoniddqd** [tttapa/Control-Surface#894] [General]
Hi, I just can't seem to communicate with my MAX7219 display using SPI.

I am wiring SPI according to [this](https://europe1.discourse-cdn.com/arduino/original/4X/2/c/e/2ce56a5776c9febb96509e941df4ffab6aa56e45.png) pinout.

MAX7219 module -> Pro micro
VCC -> VCC
GND -> GND
DN -> pin 14 (miso)
CLK - > pin 15 (sclk)
CS -> pin 10

Does anyone know if there's a special trick with Pro micro, or if there's a way to manually define DN and CLK pins instead of SPI?

```
#include <Control_Surface.h>

MAX7219SevenSegmentDisplay<> disp{SPI, 10}; // load/CS pin

void setup() {
  disp.begin();
  disp.display("hello.");
}

void loop() {}
```

**Comment by tttapa** [comment]
You're sending data from the Arduino to the MAX, so DN should be connected to MOSI, not MISO.
